adafruit_esp32spi/adafruit_esp32spi_wsgiserver.py

```diff
@@ -95,13 +95,27 @@ def update_poll(self):
         Call this method inside your main event loop to get the server
         check for new incoming client requests. When a request comes in,
         the application callable will be invoked.
+
+        Optional routine to control what IP connects to your ESP server
+        as it forces a single connection as the server can't handle more
+        than one request at a time.
+
+        self.client_available()
+        if (self._client_sock and self._client_sock.available()):
+            result = self.check_remote_ip()
+            if result == "192.168.4.2":
+                self.print_remote_ip()
```

Comment:
If you have removed the print then this line should also be removed

Reply:
Removed per request.

Reply:
The purpose was to provide new users a usage case so they might be motivated to grow with the platform.

Reply:
I had no concern with the example -- only with the line using print_remote_ip() since it was no longer present.